src/content/datachannel/datatransfer/js/main.js

```diff
@@ -7,6 +7,8 @@
  */
 
 'use strict';
+const CHUNK_SIZE = 16384;
```

Comment:
Or actually, forget about that! These examples are using adapter, so let's please use `pc.sctp.maxMessageSize`.

Reply:
That would be an implementation detail of WebRTC and not really relevant to the sample itself.

Reply:
Huh? `maxMessageSize` is part of the specification. I'm not saying to blindly accept `maxMessageSize` but something along the line of `Math.min(pc.sctp.maxMessageSize, 2**18)` to keep the chunks relatively small. This would be good practice because it removes a magic number (which people will simply copy and not understand) and instead relies on a detection mechanism that will eventually be implemented in the browsers themselves.

Reply:
Ok, `pc.sctp.maxMessageSize` could work for this sample. I'll update that.

Reply:
I'll include that in the PR too then.